packages/components/src/Drawer/Drawer.tsx

```diff
@@ -45,17 +63,21 @@ const Backdrop = styled.div<{ $isOpen: boolean }>`
 `;
 
 const Container = styled.div<{
-  anchor: Anchor;
+  $anchor: Anchor;
   $isOpen: boolean;
   $backgroundColor?: string;
 }>`
   position: fixed;
-  transition: all ease-out 0.4s;
+  transition: transform 0.2s cubic-bezier(0, 0, 0.2, 1) 0s;
```

Comment:
닫을 때는 visibility에 transition이 없어서 애니메이션이 안되더라구요ㅠㅠ 

```suggestion
  transition: transform 0.2s cubic-bezier(0, 0, 0.2, 1) 0s, visibility 0.2s cubic-bezier(0, 0, 0.2, 1) 0s;
```
요렇게 추가해주시면 돌아갈거에요!